public/service-worker.js

```diff
@@ -0,0 +1,20 @@
+// Since we migrated to Vite, the service worker strategy changed, in CRA it was a custom service worker named service-worker.js and in Vite its sw.js handled by vite-plugin-pwa
```

Comment:
you should include this guy in the `workbox.globIgnores`  entry

Reply:
yep done